lib/awful/mouse/init.lua

```diff
@@ -298,7 +298,7 @@ function mouse.object.set_current_client() end
 function mouse.object.get_current_wibox()
     local obj = capi.mouse.object_under_pointer()
     if type(obj) == "drawin" then
-        return obj
+        return obj.wibox()
```

Comment:
Please use `local wibox = obj.wibox if wibox then return wibox() end` (just in case anyone dares to use a drawin directly, without going through the wibox code)